tasks/task05/user_factory.py

```diff
@@ -0,0 +1,34 @@
+def create_user(*user, age=42, **extra):
```

Comment:
And good practice is when you use parameter types in function
Like this: `name: str, surname: str, age:int =42, **kwargs`
And bad practice is rename kwargs

Reply:
Got it!